drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilities.java

```diff
@@ -415,6 +415,10 @@ public String getBoundFactType( String boundName ) {
         return "";
     }
 
+    public GuidedDecisionTable52 getModel() {
```

Comment:
Doesn't ```DecisionTableAnalyzer``` have the ```GuidedDecisionTable52``` object set in it's constructor? Getting it from ```ColumnUtilities``` "smells wrong"; as this would have it set in it's constructor too - possibly from ```DecisionTableAnalyzer```?